### PR TITLE
Updated Query for Kubernetes blog

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/query-kubernetes/index.md [hunk 1]
```diff
@@ -2,25 +2,24 @@
 title: Introducing Pulumi Query
 authors: ["joe-duffy"]
 tags: ["Pulumi-News", "Kubernetes"]
-meta_desc: "Today we announced Pulumi Pulumi Query, an SDK for programmatically querying cloud resources."
-date: "2019-11-14"
+meta_desc: "Announcing Pulumi Pulumi Query, an SDK for programmatically querying cloud resources."
```
Comment (lukehoban) [inline content/blog/query-kubernetes/index.md]:
```suggestion
meta_desc: "Announcing Pulumi Query for Kubernetes, an SDK for programmatically querying cloud resources."
```

File: content/blog/query-kubernetes/index.md [hunk 2]
```diff
@@ -2,25 +2,24 @@
 title: Introducing Pulumi Query
```
Comment (lukehoban) [inline content/blog/query-kubernetes/index.md]:
```suggestion
title: Introducing Pulumi Query for Kubernetes
```

File: content/blog/query-kubernetes/index.md [hunk 3]
```diff
@@ -99,7 +98,7 @@ scales down the old one.
 
 For streaming queries over multiple resources, we expose a `ResourceSet`, which is a convenience
 data structure that simplifies the code. `ResourceSet` observes a sequence of resource updates
-published by `watch`, and uses those updates to keep track of which resources currently exist. It
+published by `watch`, and uses those updates to keep track of which resources currently exists. It
```
Comment (lukehoban) [inline content/blog/query-kubernetes/index.md]:
```suggestion
published by `watch`, and uses those updates to keep track of which resources currently exist. It
```

**Comment by lukehoban** [comment]
@spara The three comments I left above seem to have been resolved but not incorporated into the PR - was that on purpose?

**Comment by spara** [comment]
I missed that when I resolved the conflict changing the the authors.

**Pull request merged**